metricks.gemspec

```diff
@@ -1,16 +1,19 @@
-require File.expand_path('lib/metricks/version', __dir__)
+# frozen_string_literal: true`
 
-Gem::Specification.new do |s|
-  s.name          = 'metricks'
-  s.description   = 'An ActiveRecord backend for recording and gathering metrics'
-  s.summary       = s.description
-  s.homepage      = 'https://github.com/adamcooke/metricks'
-  s.licenses      = ['MIT']
-  s.version       = Metricks::VERSION
-  s.files         = Dir.glob('{lib,db,app}/**/*')
-  s.require_paths = ['lib']
-  s.authors       = ['Adam Cooke']
-  s.email         = ['me@adamcooke.io']
-  s.add_runtime_dependency 'activerecord', '>= 5.0'
-  s.add_runtime_dependency 'with_advisory_lock', '>= 4.6', '< 5.0'
+require_relative File.expand_path('lib/metricks/version', __dir__)
```

Comment:
I don't think we need `require_relative` here, as `File.expand_path` returns an absolute path.

Reply:
now fixed to amend `require_relative` to just `require`